project/MicroServiceBuild.scala

```diff
@@ -62,16 +47,16 @@ private object AppDependencies {
   object Test {
     def apply() = new TestDependencies {
       override lazy val test = Seq(
-        "uk.gov.hmrc" %% "hmrctest" % hmrcTestVersion % scope,
-        "org.scalatest" %% "scalatest" % scalatestVersion % scope,
-        "org.scalamock" %% "scalamock" % "3.6.0" % scope,
-        "org.scalatestplus.play" %% "scalatestplus-play" % scalatestPlusPlayVersion % scope,
-        "org.pegdown" % "pegdown" % pegdownVersion % scope,
-        "uk.gov.hmrc" %% "reactivemongo-test" % reactiveMongoTest % scope,
-        "org.reactivemongo" %% "reactivemongo-iteratees" % reactiveMongoVer,
-        "com.typesafe.akka" % "akka-testkit_2.11" % akkaContribVersion % scope, // Check it
-        "org.mockito" % "mockito-core" % mockitoCoreVersion % scope,
-        "uk.gov.hmrc" %% "tax-year" % taxyearVersion % scope
+        "uk.gov.hmrc"             %% "hmrctest"                 % "3.4.0-play-25" % scope,
+        "org.scalatest"           %% "scalatest"                % "3.0.2"         % scope,
+        "org.scalamock"           %% "scalamock"                % "3.6.0"         % scope,
+        "org.scalatestplus.play"  %% "scalatestplus-play"       % "2.0.1"         % scope,
+        "org.pegdown"             %  "pegdown"                  % "1.6.0"         % scope,
+        "uk.gov.hmrc"             %% "reactivemongo-test"       % "4.7.0-play-25" % scope,
+        "org.reactivemongo"       %% "reactivemongo-iteratees"  % "0.16.1",
+        "com.typesafe.akka"       %  "akka-testkit_2.11"        % "2.4.10"        % scope, // Check it
```

Comment:
did you check it? you can also replace %  "akka-testkit_2.11" with %% "akka-testkit"